modules/test/playwright/tests/commerce/commerce-order-content-web/pendingOrder.spec.ts

```diff
@@ -310,81 +318,292 @@ test('LPD-4174 Sales agent can receive email notifications for new orders placed
+					'MANAGE_COMMERCE_ORDER_DELIVERY_TERMS',
+					'MANAGE_COMMERCE_ORDER_PAYMENT_METHODS',
+					'MANAGE_COMMERCE_ORDER_PAYMENT_TERMS',
+					'MANAGE_COMMERCE_ORDER_SHIPPING_OPTIONS',
+					'VIEW_BILLING_ADDRESS',
+					'VIEW_COMMERCE_ORDERS',
+					'VIEW_OPEN_COMMERCE_ORDERS',
+				],
+				primaryKey: '0',
+				resourceName: 'com.liferay.commerce.order',
+				scope: 3,
+			},
+		],
+	});
+
+	await apiHelpers.headlessAdminUser.postRoleUserAccountAssociation(
+		role.id,
+		user.id
+	);
+
+	await apiHelpers.jsonWebServicesUserApiHelper.addGroupUsers(site.id, [
+		user.id,
+	]);
+
+	const option = await apiHelpers.headlessCommerceAdminCatalog.postOption(
+		'select',
+		getRandomString(),
+		'Color',
+		1
+	);
+
+	const catalog = await apiHelpers.headlessCommerceAdminCatalog.postCatalog();
+
+	const product1 = await apiHelpers.headlessCommerceAdminCatalog.postProduct({
+		catalogId: catalog.id,
+		name: {en_US: getRandomString()},
+	});
+
+	const product2 = await apiHelpers.headlessCommerceAdminCatalog.postProduct({
+		catalogId: catalog.id,
+		name: {en_US: getRandomString()},
+	});
+
+	const productBundleName = getRandomString();
+
+	const productBundle =
+		await apiHelpers.headlessCommerceAdminCatalog.postProduct({
+			catalogId: catalog.id,
+			name: {en_US: productBundleName},
+			productOptions: [
+				{
+					fieldType: 'select',
+					key: option.key,
+					name: option.name,
+					optionId: option.id,
+					priceType: 'static',
+					priority: 1,
+					productOptionValues: [
+						{
+							deltaPrice: 10.0,
+							key: 'black',
+							name: {
+								en_US: 'Black',
+							},
+							priority: 1,
+							quantity: 1,
+							skuId: product1.skus[0].id,
+						},
+						{
+							deltaPrice: 20.0,
+							key: 'white',
+							name: {
+								en_US: 'White',
+							},
+							priority: 2,
+							quantity: 1,
+							skuId: product2.skus[0].id,
+						},
+					],
+					skuContributor: true,
+				},
+			],
+		});
+
+	await applicationsMenuPage.goToProducts();
+
+	await commerceAdminProductPage.managementToolbarSearchInput.fill(
+		productBundleName
+	);
+	await commerceAdminProductPage.managementToolbarSearchInput.press('Enter');
+
+	await page
+		.getByRole('link', {exact: true, name: productBundleName})
+		.click();
+
+	await commerceAdminProductPage.generateSkus();
+
+	await expect(page.getByText('Showing 1 to 3 of 3 entries.')).toBeVisible();
+
+	const productBundleSkus = await apiHelpers.headlessCommerceAdminCatalog
+		.getProduct(productBundle.productId)
+		.then((product) => {
+			return product.skus;
+		});
+
+	const sku = productBundleSkus.find((sku) => sku.sku === 'BLACK');
+
+	try {
+		await performLogout(page);
+
+		await performLogin(page, user.alternateName);
+
+		await page.goto(
+			`${liferayConfig.environment.baseUrl}/web${site.friendlyUrlPath}/catalog`
+		);
+
+		await commerceMiniCartPage.miniCartButton.click();
+		await commerceMiniCartPage.searchProductsInput.fill(sku.sku);
```

Comment:
why don't you put the string 'BLACK' directly?

Reply:
I try to avoid using strings when possible